packages/KButton/KButton.vue

```diff
@@ -118,9 +118,15 @@ export default {
   transition: all .2s ease-in-out;
   cursor: pointer;
 
-  &:disabled {
+  &:disabled,
+  &[disabled] {
     opacity: 0.3;
     cursor: not-allowed;
+    pointer-events: none;
```

Comment:
I think we only want `pointer-events: none` for anchor tags, so that we can still get the `cursor: not-allowed;` for buttons when they are disabled.